core/domain/classifier_domain_test.py

```diff
@@ -59,7 +61,10 @@ def setUp(self):
             'algorithm_version': 1
         }
 
-    def _get_training_job_from_dict(self, training_job_dict):
+    def _get_training_job_from_dict(
+        self,
+        training_job_dict: Dict[str, Any]
```

Comment:
Shouldn't this be the `ClassifierTrainingJobDict`?

Reply:
I tried `ClassifierTrainingJobDict` here but I am getting a bunch of errors like this.

![v_2](https://user-images.githubusercontent.com/89520981/146018990-d8780998-8838-4a9e-87bd-e118b7a119fd.png)

Reply:
You might need to modify line 86 to be `expected_training_job_dict: ClassifierTrainingJobDict = {`, which should help with the error.

Reply:
done